### PR TITLE
v1.15 fix Sprintf format 'verbs' in locale files

### PR DESCRIPTION
**Pull request by axifive**
as title

### CODE REVIEW

**Comment by delvh** [comment]
If I understand the Crowdin Process correctly, this PR is unnecessary as it will be overwritten by Crowdin with the next "update translations" commit.
The best way would then be to fix it manually in Crowdin for each of these locales.

**Comment by axifive** [comment]
> If I understand the Crowdin Process correctly, this PR is unnecessary as it will be overwritten by Crowdin with the next "update translations" commit.
> The best way would then be to fix it manually in Crowdin for each of these locales.

it will be overwritten by Crowdin only in main branch.

File: options/locale/locale_de-DE.ini [hunk 1]
```diff
@@ -326,7 +326,7 @@ hi_user_x=Hallo <b>%s</b>,
 
 activate_account=Bitte aktiviere dein Konto
 activate_account.title=%s, bitte aktiviere dein Konto
-activate_account.text_1=Hallo <b>%[1]s</b>, danke für deine Registrierung bei %[2]!
+activate_account.text_1=Hallo <b>%[1]s</b>, danke für deine Registrierung bei %[2]s!
```
Comment (delvh) [inline options/locale/locale_de-DE.ini:329]: Weird.
In crowdin, both `s` are already present…
image

File: options/locale/locale_es-ES.ini [hunk 2]
```diff
@@ -326,7 +326,7 @@ hi_user_x=Hola <b>%s</b>,
 
 activate_account=Por favor, active su cuenta
 activate_account.title=%s, por favor activa tu cuenta
-activate_account.text_1=¡Hola <b>%[1]s</b>, gracias por registrarse en %[2]!
+activate_account.text_1=¡Hola <b>%[1]s</b>, gracias por registrarse en %[2]s!
```
Comment (delvh) [inline options/locale/locale_es-ES.ini:329]: Weird again:
image

**Comment by 6543** [comment]
yes crowding update only happen on main

**Comment by 6543** [comment]
was it fixed on crowdin for main?

**Review by 6543 — APPROVED** [review]

File: options/locale/locale_nl-NL.ini [hunk 3]
```diff
@@ -1048,7 +1048,7 @@ issues.action_assignee=Toegewezene
 issues.action_assignee_no_select=Geen verantwoordelijke
 issues.opened_by=%[1]s geopend door <a href="%[2]s">%[3]s</a>
 issues.closed_by=door <a href="%[2]s">%[3]s</a> gesloten %[1]s
-issues.closed_by_fake=met %[2]gesloten %[1]s
+issues.closed_by_fake=met %[2]s gesloten %[1]s
```
Comment (delvh) [inline options/locale/locale_nl-NL.ini:1051]: Now I'm completely weirded out, I cannot even find the Netherlands locale in Crowdin…

**Comment by axifive** [comment]
> was it fixed on crowdin for main?
No, Only Russian locale

File: options/locale/locale_uk-UA.ini [hunk 4]
```diff
@@ -1171,7 +1171,7 @@ issues.action_milestone_no_select=Етап відсутній
 issues.action_assignee=Виконавець
 issues.action_assignee_no_select=Немає виконавеця
 issues.opened_by=%[1]s відкрито <a href="%[2]s">%[3]s</a>
-pulls.merged_by=до <a href="%[2]s">%[3]</a> злито %[1]s
+pulls.merged_by=до <a href="%[2]s">%[3]s</a> злито %[1]s
```
Comment (delvh) [inline options/locale/locale_uk-UA.ini:1174]: Does not seem translated on Crowdin…

**Review by delvh — APPROVED** [review]
Did the rest.
You apparently took care of the russian translations.

File: options/locale/locale_uk-UA.ini (hunk 4 above)
Comment (6543) [reply options/locale/locale_uk-UA.ini:1174]: ah yes ... since it to reworded ... (english source)

File: options/locale/locale_es-ES.ini (hunk 2 above)
Comment (axifive) [reply options/locale/locale_es-ES.ini:329]: Sometimes when I translate I looking through other languages variants 🤷🏻‍♂️

File: options/locale/locale_nl-NL.ini (hunk 3 above)
Comment (axifive) [reply options/locale/locale_nl-NL.ini:1051]: https://crowdin.com/project/gitea/nl

**Comment by lunny** [comment]
It's for v1.15, so the backport label is not necessary.

File: options/locale/locale_de-DE.ini (hunk 1 above)
Comment (techknowlogick) [reply options/locale/locale_de-DE.ini:329]: They were probably changed after we made the RC

**Comment by lunny** [comment]
make L-G-T-M work.

**Review by lunny — APPROVED** [review]

**Comment by delvh** [comment]
@lunny I know.
That bot seems broken.
No matter what I do, it does not count my approvals.
I've already accepted my fate.

**Pull request merged**